jest.config.js

```diff
@@ -19,7 +19,9 @@ module.exports = {
   collectCoverage: true,
   coverageDirectory: '<rootDir>/test/unit/coverage',
   collectCoverageFrom: [
-    '<rootDir>/components/**/*.vue',
-    '<rootDir>/pages/**/*.vue',
+    '<rootDir>/src/components/**/*.vue',
+    '<rootDir>/src/layouts/**/*.vue',
+    '<rootDir>/src/pages/**/*.vue',
```

Comment:
Would `'<rootDir>/src/**/*.vue'` capture any unwanted paths?

Reply:
No, it wouldn't! I haven't thought of that, thank you :)